### DISCUSSION TITLE
Homebrew Formula for Linux

### DISCUSSION THREAD

**Discussion by mkasberg** [ghostty-org/ghostty#3295] [Feature Requests, Ideas] [answered]
I'm aware that there's already a Homebrew cask available for Ghostty on macOS. In addition, I think it would be great to create a Homebrew formula that can build Ghostty from source. This could be a good installation method on Linux for example, until official packages make it into the repositories for various distributions (which might take a little longer).

Let me know if anyone's already working on this? Otherwise, I plan on looking into creating this Homebrew formula myself and would love any tips or suggestions along the way.

**Comment by mitchellh** [comment]
Let me know if you or anyone else has any problems. Also once complete we can update the website to point to it if necessary!

**Comment by mkasberg** [comment, marked as answer]
I wanted to close the loop here. I tried to get this working but ran into several hurdles around getting all of the Ubuntu dependencies installed and configured correctly to build with Homebrew (Linuxbrew) on Ubuntu. After making slow progress on this, I wondered if it would be easier to just build a `.deb` for Ubuntu. I made much faster progress there, and I'm currently focusing on making that installation approach a better experience. https://github.com/mkasberg/ghostty-ubuntu

I do still think a (non-cask) Homebrew formula could be useful for both macOS and Linux, but I'm not currently working on it.

**Comment by Emin017** [comment]
I wrote a homebrew tap for building ghostty from source code here: https://github.com/Emin017/homebrew-ghostty

**Comment by SHU-red** [comment]
Ghostty from brew for Linux &rarr; `+1`
